content/docs/getting-started/getting-started-for-admins/index.md

```diff
@@ -71,6 +71,22 @@ By default, GitHub adds your account to the team as a member. You are now an adm
 
 ℹ️  &nbsp;It can take some time for Roadie to refresh the list of teams from GitHub teams. If you do not see admin functions immediately, please wait a few minutes and try again.
 
+### More Information on Permissions Needed
+The GitHub permissions that Roadie needs are all read only.
+
+You can also choose to allow access to all repos, or select the repos that you would like to give permission to: 
+![image](https://user-images.githubusercontent.com/42720161/127188857-bf3af155-142c-4a4e-b918-369b93823efb.png)
+
+
+**Here is a description of each of the permissions requested:**
```

Comment:
Interesting side note, is that we may need to add permissions to write when we are introducing the scaffolder.

Reply:
Interesting, I can try remember to update it then.